src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -119,6 +119,8 @@ const typeDefs = `
 
     metacardsByTag(tag: String!, settings: QuerySettingsInput): QueryResponse
     metacardsById(ids: [ID]!, settings: QuerySettingsInput): [QueryResponse]
+    metacardById(id: ID!): MetacardAttributes
```

Comment:
✏️ Would recommend keeping the response the same as the rest to stay consistent (i.e. `QueryResponse`)

Reply:
I wanted to do this but couldn't get the cacheRedirect option to play nicely with MetacardAttributes being a nested object. I felt like this wasn't that big of a compromise since most of what you'd be interested in would just be the attributes anyways